client/.eslintrc

```diff
@@ -47,7 +47,8 @@
     "jsdoc/require-property-type": "off",
     "jsdoc/require-returns-type": "off",
     "jsdoc/require-param-type": "off",
-    "jsdoc/no-undefined-types": "off"
+    "jsdoc/no-undefined-types": "off",
+    "indent": ["error", 2]
```

Comment:
This change makes building storybook from scratch fail for me. Anyone else able to reproduce?
```bash
rm -r node_modules
yarn
yarn run storybook
```

Reply:
Yes, because we have many files that are indented with 4. I've changed it to a warning so storybook should build now, though with many warnings.
11b4ae3